t/plugin/proxy-rewrite2.t

```diff
@@ -207,3 +207,26 @@ X-Forwarded-Proto: http
 X-Forwarded-Proto: grpc
 --- response_headers
 X-Forwarded-Proto: http
+
+=== TEST 7: customize X-Forwarded-Port
+--- apisix_yaml
+routes:
+  -
+    id: 1
+    uri: /echo
+    plugins:
+        proxy-rewrite:
+            headers:
+                X-Forwarded-Port: 10080
+    upstream_id: 1
+upstreams:
+  -
+    id: 1
+    nodes:
+        "127.0.0.1:1980": 1
+    type: roundrobin
+#END
+--- request
+GET /echo
```

Comment:
Would you fix the lint: https://github.com/apache/apisix/actions/runs/3375534175/jobs/5602706983 and update the test to send a X-Forwarded-Port header like https://github.com/apache/apisix/blob/03e7606b6fd0fafa707d9ada6d6318c2e5c30bd8/t/plugin/proxy-rewrite3.t#L334?

Reply:
Whether two different case are required, one is  the X-Forwarded-Port header in  original request, another is in plugin proxy-rewrite.